evergreen/run-tests.sh

```diff
@@ -102,26 +102,45 @@ if [[ "$OS" =~ Windows|windows ]]; then
     certutil.exe -urlcache localhost delete # clear the OS-level cache of all entries with the URL "localhost"
   fi
 else
-  export TARGET="Test"
+  if [[ "$FRAMEWORK" == "nil" ]]; then
+    export TARGET="Test"
+  else
+    export TARGET="Test${FRAMEWORK}"
+  fi
```

Comment:
If `$FRAMEWORK` is `nil`, then `"Test${FRAMEWORK}"` equals `Test`. Then again, explicitly calling out that `$FRAMEWORK` could be `nil` is more obvious. Up to you whether you'd prefer the above or the terser:
`export TARGET="Test${FRAMEWORK}"`

Reply:
Changed per your suggestion.